app/src/test/java/src/ShapeITest.java

```diff
@@ -8,9 +8,30 @@
 
     @Test
     public void draw_DrawingShapeI_ArrayOfNumbers() {
-        ShapeI shapeI = new ShapeI();
+        ShapeI shapeI = new ShapeI(1, 1);
         int[][] actual = shapeI.draw();
         int[][] expected = {{ 1, 1, 1, 1}};
         assertArrayEquals(expected, actual);
     }
+
+    @Test
+    public void itShouldMoveRight() {
```

Comment:
joel the names of the tests are not as we define them

Reply:
You're right. I already changed them.

Reply:
Yes now they are fine thanks Joel